catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableXmlElementImpl.java

```diff
@@ -464,4 +468,49 @@ private LiteralElement(JAXBElement element) {
       return value;
     }
   }
+
+  /**
+   * NB: This should only be called when {@code visitable} is guaranteed to contain a {@link
```

Comment:
:question: What does "NB:" mean? Is this an established or documented convention? If not, I would like to see it replaced with something more meaningful.

Reply:
https://en.wikipedia.org/wiki/Nota_bene

It's been an established convention since the year 1711 :-)

Reply:
Fair enough.